lib/policies/load-balancing.js

```diff
@@ -167,6 +169,14 @@ DCAwareRoundRobinPolicy.prototype.newQueryPlan = function (keyspace, queryOption
   var self = this;
   var counter = 0;
   var remoteCounter = 0;
+
```

Comment:
Hrmm, I don't imagine this being easy without filtering out local and remote hosts into separate arrays at the beginning of the query plan or doing some refactoring of load balancing policies so DC hosts can be maintained separately from one another.  My concern about creating arrays for each query plan can be computationally expensive when you have a large number of hosts.

Thoughts @jorgebay?